src/UI/Settings/General/GeneralViewTemplate.hbs

```diff
@@ -162,6 +162,70 @@
             </div>
         </div>
     </fieldset>
+	
+	<fieldset>
+        <legend>Proxy Settings</legend>
+		
+		<div class="form-group">
+            <label class="col-sm-3 control-label">Use Proxy</label>
+
+			<div class="col-sm-1 col-sm-push-4 help-inline">
+                <i class="icon-sonarr-form-warning" title="Requires restart to take effect"/>
```

Comment:
I don't see anything in the code that make it seem it actually requires a restart, unless I'm missing something.